src/main/scala/akka/persistence/redis/RedisUtils.scala

```diff
@@ -18,40 +18,61 @@ package persistence
 package redis
 
 import actor._
-
 import _root_.redis._
-
+import akka.persistence.utils.HostAndPort
 import com.typesafe.config.Config
 
 import scala.collection.JavaConverters._
 
 object RedisUtils {
 
-  def host(conf: Config) = conf.getString("redis.host")
+  def host(conf: Config): String = conf.getString("redis.host")
+
+  def port(conf: Config): Int = conf.getInt("redis.port")
 
-  def port(conf: Config) = conf.getInt("redis.port")
+  def database(conf: Config): Option[Int] =
+    if (conf.hasPath("redis.database")) Some(conf.getInt("redis.database"))
+    else None
 
-  def database(conf: Config) = if (conf.hasPath("redis.database")) Some(conf.getInt("redis.database")) else None
+  def password(conf: Config): Option[String] =
+    if (conf.hasPath("redis.password")) Some(conf.getString("redis.password"))
+    else None
 
-  def password(conf: Config) = if (conf.hasPath("redis.password")) Some(conf.getString("redis.password")) else None
+  def sentinels(conf: Config): Option[Seq[(String, Int)]] =
+    conf.getString("redis.mode") match {
+      case "sentinel" =>
```

Comment:
This has already been checked when this method is called, it is not necessary. The method will only be called in such cases, and it can simply return a `Seq[(String, Int)]`.

Maybe put an assertion at the beginning of the method to check it.